src/ParallelRun.jl

```diff
@@ -136,21 +141,21 @@ function infer_init(rcfs::Vector{RunCamcolField},
     tic()
     catalog = SDSSIO.read_photoobj_files(rcfs, stagedir,
                         duplicate_policy=duplicate_policy)
-    timing.read_photoobj = toq()
-    Log.info("$(length(catalog)) primary sources")
+    timing.read_photoobj = timing.read_photoobj + toq()
+    #ntputs(grank(), threadid(), "$(length(catalog)) primary sources")
```

Comment:
If you don't want the info commands printed, rather than comment these out, would you change Log.info to only print when the CELESTE_PROD environment variable is not set? I did that for Log.debug already.